swift-ticketing/src/swift_ticketing/db/booking.clj

```diff
@@ -1,25 +1,34 @@
 (ns swift-ticketing.db.booking
-  (:require [honey.sql :as sql]))
+  (:require [honey.sql :as sql]
+            [swift-ticketing.db.query :refer [run-query]]))
 
 (defonce INPROCESS "InProcess")
 (defonce CONFIRMED "Confirmed")
 (defonce PAYMENTPENDING "PaymentPending")
 (defonce REJECTED "Rejected")
 (defonce CANCELED "Canceled")
 
-(defn insert-booking [uid booking-id]
+(defn insert-booking-query [uid booking-id]
   (sql/format {:insert-into :booking
-                 :columns [:booking_id :user_id :booking_status]
-                 :values [[booking-id
-                           [:cast uid :uuid]
-                           [:cast INPROCESS :booking_status]]]}))
+               :columns [:booking_id :user_id :booking_status]
+               :values [[booking-id
+                         [:cast uid :uuid]
+                         [:cast INPROCESS :booking_status]]]}))
 
-(defn get-booking-status [uid booking-id]
+(defn insert-booking [db-spec & args]
+  (run-query db-spec insert-booking-query args))
+
+(defn get-booking-status-query [uid booking-id]
```

Comment:
You can use a `select *` here. Selecting specific columns is an optimization for much later with the trade-off that it makes the functions less re-usable.